plugins/modules/sl_vm.py

```diff
@@ -138,11 +138,11 @@
     type: int
   public_vlan:
     description:
-      - VLAN by its Id to be assigned to the public NIC.
+      - VLAN by its ID to be assigned to the public NIC.
```

Comment:
There are way too many instances of this as `id` as well. I can tweak andebox to look for that as well, but I wasn't clear about a "standard" to adopt. Should it always be upper case?

Reply:
I think it should always be `ID` (upper-case).

Reply:
Okie, I'll put it on the radar, but not entirely sure the upper-case is going to work in every situation.